doc/user-guide.adoc

```diff
@@ -477,6 +478,17 @@ Users can turn off the auto-detection mechanism by setting the `bindings.autoDet
 
 _This feature is only available if you have Service Binding Operator installed on your cluster._
 
+==== Exposing `RuntimeComponent` applications as Provisioned Services
+
+A `RuntimeComponent` application can be configured to behave as a link:++https://github.com/k8s-service-bindings/spec#provisioned-service++[Provisioned Service] defined by the link:++https://github.com/k8s-service-bindings/spec++[Service Binding Specification].
+
+According to the specification, a Provisioned Service resource must define a `.status.binding.name` which is a `LocalObjectReference`-able to a Secret.
```

Comment:
whatever thing "LocalObjectReference" is, put a noun for that thing after "LocalObjectReference". In googling, I see that it is a class. Reword "which is a `LocalObjectReference`-able to a Secret." to something like "which is a `LocalObjectReference`-class. This class referenceable to a Secret.".